### ISSUE TITLE
Art: re-organize Google Drive

### ISSUE THREAD

**Issue by avaer** [webaverse/app#1464]

**Comment by avaer** [comment]
Additionally, I've been running into several cases where I could not find the model in the GDrive at all (such as the silk blob `.blend` file).

**Comment by Vianvolaeus** [comment]
That's an older proof of concept file early from #1338, which was marked as complete, but it's not clear which is the correct file. 

I'm going through and filling out descriptor boxes on my own assets since that's fast, but a screenshot pass is definitely a good idea.

Generally speaking I've been filing things as follows, since it seems to make the most sense going from Github issue to direct file:

Assignee (creator) > category > item > folder with most recent iteration, and archive of older files.

**Comment by avaer** [comment]
> Assignee (creator) > category > item > folder with most recent iteration, and archive of older files.

Yes, that seems like the best approach. I think it's the responsibility of each assignee to organize their folders.

**Comment by avaer** [comment]
Re: storage, we can re-own the directories/folders to Webaverse org so it's not a burden on anyone's account.

**Comment by Vianvolaeus** [comment]
Been working on arranging, adding descriptions and screenshots to my drive. 

I've been uploading using the image links from github directly - makes the most sense in terms of grabbing the most recent screenshot IMO.

**Comment by avaer** [comment]
> Been working on arranging, adding descriptions and screenshots to my drive.

Thank you!

> I've been uploading using the image links from github directly - makes the most sense in terms of grabbing the most recent screenshot IMO.

Yeah, that makes sense, as long as the file hasn't really changed since then.

**Comment by avaer** [comment]
This was not finished: https://github.com/webaverse/app/issues/1892
